embed.js

```diff
@@ -136,10 +185,9 @@ function embedCodeToTarget(targetDiv, codeText, showBorder, showLineNumbers, sho
   code.classList.add(lang);
   if (startLine > 0) {
     codeTextSplit = codeText.split("\n");
-    code.textContent = codeTextSplit.slice(startLine - 1, endLine).join("\n");
-  } else {
-    code.textContent = codeText;
+    codeText = codeTextSplit.slice(startLine - 1, endLine).join("\n");
   }
+  code.textContent = codeText;
```

Comment:
Using the codeText variable for the "copy" action, and the closure will keep the variable in the correct scope in the case of multiple embed.js examples on a single page.